app/main.py

```diff
@@ -16,4 +18,21 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+sorted_variables = {"mutable": [], "immutable": []}
+
+def sort_variables(*args) -> None:
+
+    for current_variable in args:
+        what_type = str(type(current_variable))
+        key = "immutable"
+
+        if "dict" in what_type or "list" in what_type or "set" in what_type:
+            key = "mutable"
+
+        sorted_variables[key].append(current_variable)
+
```

Comment:
The way you are checking the type of a variable is not very reliable and can lead to unexpected results. It would be better to use the `isinstance()` function to check the type of a variable. This function is more reliable and easier to read.